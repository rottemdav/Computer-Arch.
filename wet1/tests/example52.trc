8 1 25 3 local_history global_tables not_using_share
0x8f0 T 0xd00c
0xb9128 T 0xd384
0x302f12c8 N 0x1a265b0
0xb0a4 T 0x33b9848
0x853d8 N 0xd0c
0x010c N 0xe20fbabc
0xaa64f78 N 0xb826bac
0x4c5c N 0xf64fc0
0x9c0 N 0x13840
0xa5c T 0x50e264
0x578855bc T 0xe14
0x090 N 0x4638
0x302f12c8 N 0x1a265b0
0xca70 T 0x8194
0xa04 T 0xf8cf12b8
0x4c5c N 0xf64fc0
0x4bcfc T 0x3fc
0x578855bc T 0xe14
0x2d4 T 0x473d104
0x4bcfc T 0x3fc
0x414f808 N 0xf26e5db8
0x2b7ba84 T 0xa0f74f8
0x884 N 0x22f54
0x13eb5d8c T 0x65867870
0x9c0 T 0x13840
0x7524 N 0xa1cfc
0xca70 N 0x8194
0xa051bf8 T 0xf6debbe8
0xca70 N 0x8194
0xa04 N 0xf8cf12b8
0xd00 T 0x9fc
0x972c T 0x020e886c
0x929e3fc N 0x49bf0
0xa5c N 0x50e264
0x7182b40 N 0xacc
0x853d8 N 0xd0c
0x578855bc N 0xe14
0xa04 T 0xf8cf12b8
0x010c N 0xe20fbabc
0x1e4 N 0x334c8
0x090 T 0x4638
0x8f0 N 0xd00c
0x4b170 N 0x315a34c
0x884 T 0x22f54
0x929e3fc N 0x49bf0
0x972c T 0x020e886c
0x010c T 0xe20fbabc
0x010c T 0xe20fbabc
0xb131ab4 T 0x39efc
0xb0a4 T 0x33b9848
0x414f808 T 0xf26e5db8
0x4bcfc N 0x3fc
0x929e3fc T 0x49bf0
0x802c T 0xb5bc
0xa5c T 0x50e264
0x2128 N 0x4f424f80
0xc8cf5b98 N 0xb692600
0xa051bf8 T 0xf6debbe8
0x802c T 0xb5bc
0xb9128 T 0xd384
0xd629a7c T 0x05c4
0x2b7ba84 N 0xa0f74f8
0xfba5808 N 0xf496cd4
0x090 N 0x4638
0x884 N 0x22f54
0x4bcfc T 0x3fc
0xd1aecf4 T 0xb1cc
0x59e82484 T 0x6fa5fc1c
0xd00 N 0x9fc
0x2b7ba84 T 0xa0f74f8
0x4c5c T 0xf64fc0
0xb9128 T 0xd384
0xa051bf8 N 0xf6debbe8
0x4bcfc N 0x3fc
0x972c N 0x020e886c
0x2b7ba84 N 0xa0f74f8
0x7182b40 N 0xacc
0x7182b40 T 0xacc
0x2d4 N 0x473d104
0x802c T 0xb5bc
0x929e3fc T 0x49bf0
0xd629a7c T 0x05c4
0x8f0 N 0xd00c
0x302f12c8 N 0x1a265b0
0xb131ab4 N 0x39efc
0x8f0 N 0xd00c
0xa04 N 0xf8cf12b8
0x414f808 T 0xf26e5db8
0x853d8 T 0xd0c
0x090 T 0x4638
0x802c T 0xb5bc
0x090 N 0x4638
0x960 N 0x85f543a0
0x9c0 T 0x13840
0x7524 N 0xa1cfc
0xa04 N 0xf8cf12b8
0xb131ab4 N 0x39efc
0x884 T 0x22f54
0xb131ab4 N 0x39efc
0xb131ab4 T 0x39efc
0x802c N 0xb5bc
0xb131ab4 N 0x39efc
0xc8cf5b98 N 0xb692600
0x010c T 0xe20fbabc
0xa051bf8 N 0xf6debbe8
0x414f808 T 0xf26e5db8
0x8f0 T 0xd00c
0xb0a4 N 0x33b9848
0x9c0 T 0x13840
0x090 N 0x4638
0xa5c N 0x50e264
0x972c T 0x020e886c
0x414f808 N 0xf26e5db8
0x090 N 0x4638
0x2d4 T 0x473d104
0xaa64f78 T 0xb826bac
0xb0a4 N 0x33b9848
0x8f0 T 0xd00c
0x853d8 T 0xd0c
0xb9128 T 0xd384