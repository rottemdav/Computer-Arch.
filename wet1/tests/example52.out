0x8f0 N 0x8f4
0xb9128 N 0xb912c
0x302f12c8 N 0x302f12cc
0xb0a4 N 0xb0a8
0x853d8 N 0x853dc
0x10c N 0x110
0xaa64f78 N 0xaa64f7c
0x4c5c N 0x4c60
0x9c0 N 0x9c4
0xa5c N 0xa60
0x578855bc N 0x578855c0
0x90 N 0x94
0x302f12c8 N 0x302f12cc
0xca70 N 0xca74
0xa04 N 0xa08
0x4c5c N 0x4c60
0x4bcfc N 0x4bd00
0x578855bc N 0x578855c0
0x2d4 N 0x2d8
0x4bcfc N 0x4bd00
0x414f808 N 0x414f80c
0x2b7ba84 N 0x2b7ba88
0x884 N 0x888
0x13eb5d8c N 0x13eb5d90
0x9c0 T 0x13840
0x7524 N 0x7528
0xca70 T 0x8194
0xa051bf8 N 0xa051bfc
0xca70 T 0x8194
0xa04 N 0xa08
0xd00 N 0xd04
0x972c N 0x9730
0x929e3fc N 0x929e400
0xa5c N 0xa60
0x7182b40 N 0x7182b44
0x853d8 N 0x853dc
0x578855bc N 0x578855c0
0xa04 N 0xa08
0x10c N 0x110
0x1e4 N 0x1e8
0x90 N 0x94
0x8f0 N 0x8f4
0x4b170 N 0x4b174
0x884 N 0x888
0x929e3fc N 0x929e400
0x972c N 0x9730
0x10c N 0x110
0x10c T 0xe20fbabc
0xb131ab4 N 0xb131ab8
0xb0a4 N 0xb0a8
0x414f808 T 0xf26e5db8
0x4bcfc N 0x4bd00
0x929e3fc N 0x929e400
0x802c N 0x8030
0xa5c N 0xa60
0x2128 N 0x212c
0xc8cf5b98 N 0xc8cf5b9c
0xa051bf8 N 0xa051bfc
0x802c T 0xb5bc
0xb9128 N 0xb912c
0xd629a7c N 0xd629a80
0x2b7ba84 N 0x2b7ba88
0xfba5808 N 0xfba580c
0x90 N 0x94
0x884 N 0x888
0x4bcfc N 0x4bd00
0xd1aecf4 N 0xd1aecf8
0x59e82484 N 0x59e82488
0xd00 N 0xd04
0x2b7ba84 N 0x2b7ba88
0x4c5c N 0x4c60
0xb9128 N 0xb912c
0xa051bf8 T 0xf6debbe8
0x4bcfc N 0x4bd00
0x972c N 0x9730
0x2b7ba84 T 0xa0f74f8
0x7182b40 N 0x7182b44
0x7182b40 N 0x7182b44
0x2d4 N 0x2d8
0x802c N 0x8030
0x929e3fc N 0x929e400
0xd629a7c N 0xd629a80
0x8f0 N 0x8f4
0x302f12c8 N 0x302f12cc
0xb131ab4 N 0xb131ab8
0x8f0 N 0x8f4
0xa04 N 0xa08
0x414f808 N 0x414f80c
0x853d8 N 0x853dc
0x90 N 0x94
0x802c N 0x8030
0x90 T 0x4638
0x960 N 0x964
0x9c0 N 0x9c4
0x7524 N 0x7528
0xa04 N 0xa08
0xb131ab4 N 0xb131ab8
0x884 N 0x888
0xb131ab4 N 0xb131ab8
0xb131ab4 N 0xb131ab8
0x802c N 0x8030
0xb131ab4 N 0xb131ab8
0xc8cf5b98 N 0xc8cf5b9c
0x10c N 0x110
0xa051bf8 N 0xa051bfc
0x414f808 N 0x414f80c
0x8f0 N 0x8f4
0xb0a4 N 0xb0a8
0x9c0 N 0x9c4
0x90 N 0x94
0xa5c N 0xa60
0x972c N 0x9730
0x414f808 T 0xf26e5db8
0x90 N 0x94
0x2d4 N 0x2d8
0xaa64f78 N 0xaa64f7c
0xb0a4 T 0x33b9848
0x8f0 N 0x8f4
0x853d8 N 0x853dc
0xb9128 N 0xb912c
flush_num: 63, br_num: 120, size: 460b
